Content.Tests/DMProject/Tests/Const/Const6.dm

```diff
@@ -0,0 +1,10 @@
+﻿/proc/RunTest()
+	var/a = 137
+	switch(a)
+		if(20)
+			. = 500
+		if(136 | 1)
+			. = 1
+		if(/datum, /mob)
+			. = 300
+	ASSERT(a == 137)
```

Comment:
This should do `ASSERT(. == 1)` because this aims to verify that `if(136 | 1)` is executing. `a` is not meant to ever change.
```suggestion
	ASSERT(. == 1)
```